lib/pengine/unpack.c

```diff
@@ -402,14 +402,16 @@ unpack_config(xmlNode * config, pe_working_set_t * data_set)
     data_set->placement_strategy = pe_pref(data_set->config_hash, "placement-strategy");
     crm_trace("Placement strategy: %s", data_set->placement_strategy);
 
-    set_config_flag(data_set, "shutdown-lock", pe_flag_shutdown_lock);
-    crm_trace("Resources will%s be locked to cleanly shut down nodes",
-              (pcmk_is_set(data_set->flags, pe_flag_shutdown_lock)? "" : " not"));
-    if (pcmk_is_set(data_set->flags, pe_flag_shutdown_lock)) {
+    set_config_flag(data_set, "shutdown-lock", pcmk_sched_shutdown_lock);
+    if (pcmk_is_set(data_set->flags, pcmk_sched_shutdown_lock)) {
         value = pe_pref(data_set->config_hash,
                         XML_CONFIG_ATTR_SHUTDOWN_LOCK_LIMIT);
         data_set->shutdown_lock = crm_parse_interval_spec(value) / 1000;
-        crm_trace("Shutdown locks expire after %us", data_set->shutdown_lock);
+        crm_trace("Resources will be locked to cleanly shut down nodes "
```

Comment:
hmm ... just a trace but sounds to me as if not using shutdown locks would be something dirty ...

Reply:
maybe "... to nodes that were cleanly shut down"

Reply:
Of course - that was the point - the sentence can be read in two ways and your suggestion removes that ambiguity.